docs/antctl.md

```diff
@@ -52,3 +52,199 @@ or Agent.
 When running out-of-cluster ("controller mode" only), antctl will look for your
 kubeconfig file at `$HOME/.kube/config` by default. You can select a different
 one with `--kubeconfig`.
+
+The following sub-sections introduce a few commands which are useful for
+troubleshooting the Antrea system.
+
+### `controllerinfo` and `agentinfo` commands
+`antctl` controller command `get controllerinfo` (or `get ci`) and agent command
+`get agentinfo` (or `get ai`) print runtime information of `antrea-controller`
+and `antrea-agent` respectively.
+```
+antctl get controllerinfo
+antctl get agentinfo
+```
+
+### NetworkPolicy commands
+Both Antrea Controller and Agent support querying NetworkPolicy objects.
+- `antctl` `get networkpolicy` (or `get netpol`) command can print all
+NetworkPolicies, a specified NetworkPolicy, or NetworkPolicies in a specified
+Namespace.
+- `get appliedtogroup` (or `get atg`) command can print all NetworkPolicy
+AppliedToGroups (AppliedToGroup includes the Pods to which a NetworkPolicy is
+applied), or a specified AppliedToGroups.
+- `get addressgroup` (or `get ag`) command can print all NetworkPolicy
+AddressGroups (AddressGroup defines source or destination addresses of
+NetworkPolicy rules), or a specified AddressGroup.
+
+Using the `json` or `yaml` antctl output format can print more information of
+NetworkPolicy, AppliedToGroup, and AddressGroup, than using the default `table`
+output format.
+```
+antctl get networkpolicy [name] [-n namespace] [-o yaml]
+antctl get appliedtogroup [name] [-o yaml]
+antctl get addressgroup [name] [-o yaml]
+```
+
+Antrea Agent extrally supports printing NetworkPolicies applied to a specified
+local Pod using this `antctl` command:
+```
+antctl get networkpolicy -p pod -n namespace
+```
+
+### Dumping Pod network interface information
+`antctl` agent command `get podinterface` (or `get pi`) can dump network
+interface information of all local Pods, or a specified local Pod, or local Pods
+in the specified Namespace, or local Pods matching the specified Pod name.
+```
+antctl get podinterface [name] [-n namespace]
+```
+
+### Dumping OVS flows
+Starting from version 0.6.0, Antrea Agent supports dumping Antrea OVS flows. The
+`antctl` `get ovsflows` (or `get of`) command can dump all OVS flows, flows
+added for a specified Pod, or flows added to realize a specified NetworkPolicy,
+or flows in a specified OVS flow table.
+```
+antctl get ovsflows
+antctl get ovsflows -p pod -n namespace
+antctl get ovsflows --networkpolicy networkpolicy -n namespace
+antctl get ovsflows -T table
+```
+
+An OVS flow table can be specified using the table name or the table number.
+`antctl get ovsflow --help` lists all Antrea flow tables. For more information
+about Antrea OVS pipeline and flows, please refer to the [OVS pipeline doc](/docs/ovs-pipeline.md).
+
+Example outputs of dumping Pod and NetworkPolicy OVS flows:
```

Comment:
I hope it won't be too tedious to keep the examples up to date

Reply:
I feel it is ok, even it is not up-to-date. Probably more important to update ovs-pipeline.md.